pom.xml

```diff
@@ -41,7 +41,12 @@
     <commons-io.version>2.7</commons-io.version>
     <grpc.version>1.42.1</grpc.version>
     <immutables.version>2.8.8</immutables.version>
-    <jackson.version>2.12.6</jackson.version>
+    <!-- 28-Mar-2022, tatu: BOM for "micro-patch" 2.12.6.1 of jackson-databind
+        is bit unusual wrt version; normally would be "x.y.z": when updating see:
+          https://mvnrepository.com/artifact/com.fasterxml.jackson/jackson-bom
+        for available versions
+      -->
+    <jackson.version>2.12.6.20220326</jackson.version>
```

Comment:
No; just trying to rush this through immediately for deadline, and also since the CNDB upgrade is targeting 2.12.
I am not aware of specific problems with 2.13(.2) so worth following up with that.